languages/common-lisp/reference/exercise-concepts/anagram.md

```diff
@@ -0,0 +1,20 @@
+# Concepts of Anagram
+
+While there are, of course, different ways to attack this problem the
+'canonical' way is to write a function that "normalizes" the
+candidates and the input word so that they can then be easily
+compared. It is this method which was reviewed for concepts. (ref. the
+[exercism v2 common-lisp track exercise][v2-exercise].)
+
+## Concepts
+
+- Sequences
+- Strings
+- Higher-order functions
+- Equality
+- Sorting
```

Comment:
This feels a bit more specific / smaller than the other concepts, but it certainly could still be it's own thing if we explore the use of the `:key` function.

Reply:
... not to mention the `:test` keyword.

But I think given our discussion of 'Sequences' above - Sorting is part of that.